include/drivers/adc.h

```diff
@@ -134,6 +153,41 @@ struct adc_channel_cfg {
 #endif /* CONFIG_ADC_CONFIGURABLE_INPUTS */
 };
 
+/**
+ * @brief Convert a raw ADC value to millivolts.
+ *
+ * This function performs the necessary conversion to transform a raw
+ * ADC measurement to a voltage in millivolts.
+ *
+ * @param ref_mv the reference voltage used for the measurement, in
+ * millivolts.  This may be from adc_ref_internal() or a known
+ * external reference.
+ *
+ * @param gain the ADC gain configuration used to sample the input
+ *
+ * @param resolution the resolution used to sample the input
```

Comment:
if input is differential then (resolution - 1) should be used. Or add comment here to let user know or differential flag as function parameter.

Reply:
The documentation on resolution for differential sampling is notably lacking.  Looks like MCUX might accept either `N` or `N+1`.  I've reworded this to provide a clue, but I don't think there's a way to specify exactly what the user needs to do.